aerospike_helpers/operations/bitwise_operations.py

```diff
@@ -80,7 +80,7 @@
     from aerospike import exception as e
     from aerospike_helpers.operations import bitwise_operations
 
-    config = {'hosts': [('127.0.0.1', 3000)]}
+    config = self.connection_config.copy()
```

Comment:
```suggestion
    config = {'hosts': [('127.0.0.1', 3000)]}
```
The old line should be correct because there's no `connection_config` in the example.